token-metadata/js/src/instruction.ts

```diff
@@ -0,0 +1,53 @@
+import type { PublicKey } from '@solana/web3.js';
+
+/** The field can be one of the required fields (name, symbol, URI), or a
+ * totally new field denoted by a "key" string. */
+type Field = 'name' | 'symbol' | 'uri' | string;
```

Comment:
I followed the rust implementation, but would personally just type it as "string"

Reply:
It might seem gratuitous to include in the type definition, but it does provide a sense of reminder/security to users. Essentially, we want to make it abundantly clear that `'name' | 'symbol' | 'uri'` are fundamental parts of the interface, while any other key is additional.

This might come in handy if you're typing out a function that will attempt to update one of these keys, or erroneously trying to delete them.

Reply:
Okay follow your rationale, lets stick with how it is. Thanks for the explanation